modules/serverless-logic-data-index-graphql-queries.adoc

```diff
@@ -0,0 +1,218 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-supporting-services/serverless-logic-data-index
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-data-index-graphql-queries_{context}"]
+= GraphQL queries for workflow instances and jobs
+
+To retrieve data about workflow instances and jobs, you can use GraphQL queries.
+
+[id="serverless-logic-retrieve-data-workflow-instances_{context}"]
+== Retrieve data from workflow instances
+
+You can retrieve information about a specific workflow instance, by using the following query example:
```

Comment:
@xenolinux and @kaldesai Minor nit, the comma before `by` is not required and original edit was correct
```suggestion
You can retrieve information about a specific workflow instance by using the following query example:
```

Reply:
Right! My bad :(